srcpkgs/python3-pytest-xdist/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'python3-pytest-xdist'
+pkgname=python3-pytest-xdist
+version=2.5.0
+revision=1
+wrksrc="pytest-xdist-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip python3-wheel"
+depends="python3-pytest"
+checkdepends="${depends} tox python3-packaging python3-attrs python3-toml python3-pluggy python3-parsing"
+short_desc="Pytest plugin for distributed testing"
+maintainer="Tim Sandquist <tim.sandquist@gmail.com>"
+license="MIT"
+homepage="https://github.com/pytest-dev/pytest-xdist/"
+distfiles="${PYPI_SITE}/p/pytest-xdist/pytest-xdist-${version}.tar.gz"
+checksum=4580deca3ff04ddb2ac53eba39d76cb5dd5edeac050cb6fbc768b0dd712b4edf
+
+post_install() {
+	vlicense LICENSE
+}
+
+# adding do_check since the tests depend on this package
+do_check() {
+	python setup.py install
+	pytest
+}
```

Comment:
I don't think this is necessary, does the default do_check not work?

Reply:
Correct. Checks work locally but not in CI. This was my attempt to get it working. It looked like pytest tests had more configuration options.

Reply:
I added this because it appeared that pytest needed the xdist plugin installed to perform the tests.  It seemed odd but this solved the issue.  I removed it.  It worked locally.  We'll see if it works in CI.